DiagnosticsExtension/Controllers/ConnectionStringValidationController.cs

```diff
@@ -7,41 +7,46 @@
 
 using System;
 using System.Collections.Generic;
+using System.IO;
 using System.Linq;
 using System.Net;
 using System.Net.Http;
 using System.Net.Sockets;
 using System.Text;
 using System.Threading.Tasks;
+using System.Web;
 using System.Web.Http;
+using DiagnosticsExtension.Models;
 using DiagnosticsExtension.Models.ConnectionStringValidator;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 
 namespace DiagnosticsExtension.Controllers
 {
-    /// <summary>
-    /// Worker instances are running an udp echo server on port 30000. This controller is for checking the connection between target 
-    /// worker instance by pinging and checking the echoed result.
-    /// </summary>
     [RoutePrefix("api/connectionstringvalidation")]
     public class ConnectionStringValidationController : ApiController
```

Comment:
I see this controller was added with the PR https://github.com/Azure/DaaS/pull/11 and I never got a chance to review that one. I am trying to understand why there was a need to add this new controller and why we couldn't have just extended the existing DatabaseTestController. Right now, I see two controllers doing similar stuff.

Reply:
Yes, the two controllers are doing similar connection validation, but the controller actions are quite different, as we need the ability to call the api to test one arbitrary string read from appsetting or user input. This decides that we have to implement a different interface. And also, the name of `DatabaseTester` no longer correctly describes the functionality of KeyVault, storage account, eventhub (etc.) testing

Reply:
So I tried to re-use the code in the way of sharing common functions

Reply:
Chiming in here: @puneetg1983 it would be good to consolidate the logic into a single controller but as @yifguoMSFT highlighted, the scope of what we are doing for network validation is quite different.  Please advise if the DatabaseTester can be renamed and refactored without impacting existing workflows and if that is something we should do along with these additions.

Reply:
We cannot rename the DatabaseTester controller as its been coded in Diagnose and Solve UI for now. I understand why a need for a new Controller now.